WcaOnRails/spec/spec_helper.rb

```diff
@@ -84,4 +84,15 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  config.before(:suite) do
+    %w(Countries Continents Events Rounds Formats teams).each do |table|
+      # -- Uncomment to regenerate --
+      # ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+
+      if ActiveRecord::Base.connection.execute("SELECT COUNT(*) FROM #{table}").first == [0]
+        load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
Yeah it's really weird to me as well.
Very good point with seed files changes but I'm not sure how could we deal with this.